docs/version0.15.md

```diff
@@ -69,6 +70,17 @@ For compatibility, the [`-XX:OnOutOfMemoryError`](xxonoutofmemoryerror.md) OpenJ
 
 The VM now supports the allocation of huge pages on Linux when you use the `madvise` (`/sys/kernel/mm/transparent_hugepage/enabled`) setting. To enable this feature, set [`-XX:+TransparentHugePage`](xxtransparenthugepage.md) on the command line when you start your application. This option is currently not enabled by default.
 
+###  ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) Support for low-overhead heap profiling
+
+[JEP 331](http://openjdk.java.net/jeps/331) provides a mechanism for sampling Java heap allocations with a low overhead via
+the JVM Tool Interface (JVMTI).
+
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Restrictions:** JEP 331 is implemented for OpenJ9 with the following limitations:
+
+- The `balanced` and `metronome` garbage collection policies are not supported.
+- Bu default, a JEP 331 JVMTI agent uses the same sampling interval as the Health Center agent. If both agents expect the same interval
```

Comment:
Bu -> By

Reply:
Just checking you were awake ;-) ...that's gone now anyway. Holding off for a few more minutes to let the conversation come to a close.